ui/src/app/shared/components/revision.tsx

```diff
@@ -5,7 +5,13 @@ export const Revision = ({repoUrl, revision, children}: {repoUrl: string; revisi
     revision = revision || '';
```

Comment:
Did you mean to change `revision.tsx` in this PR?

Reply:
I didn't, must be an issue with my linter